### PR TITLE
Build on base landing

### PR DESCRIPTION
**Pull request by shift4id**

### CODE REVIEW

**Comment by jnsdls** [comment]
#### 📝 Changed routes:


| [**/build/base**](https://thirdweb-fra5xh5ko.thirdweb-preview.com/build/base) |
|:-:|
|<a href="https://thirdweb-fra5xh5ko.thirdweb-preview.com/build/base">Screenshot of /build/base</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/MZz4_t3lArtqfJ27x0ZaJv5rokV-T0SSNIfqVRqgDS96Q2f_bMrzVlUxdRmARPZY4MThj8NLCmmLgXpkXKZSHw.png">(view full size)</a> |







Commit deea09bb481365a5eed1d63644853934ac6eeb05 (https://thirdweb-fra5xh5ko.thirdweb-preview.com).

**Comment by nachoiacovino** [comment]
shouldn't the URL be /network/base so we have the same structure as /network/solana?

**Comment by jnsdls** [comment]
> shouldn't the URL be /network/base so we have the same structure as /network/solana? or is it SEO reasons?

no, it's not a network page

File: pages/build/base.tsx [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+                      letterSpacing="-0.02em"
+                      background="linear-gradient(80deg, #6891F7, #C0D2FF, #6891F7)"
+                      backgroundClip="text"
+                      backgroundSize="200% 200%"
+                      transitionProperty="background-position, transform"
+                      transitionDuration="slower"
+                      _hover={{
+                        backgroundPosition: "bottom right",
+                      }}
+                      {...{ as: LinkOverlay, href, isExternal: true }}
```
Comment (nachoiacovino) [inline pages/build/base.tsx:338]: what's the reason for this being a destructured object?

**Pull request merged**